logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -11,6 +11,7 @@ module LogStash module Inputs
   # Elasticsearch
   class Metrics < LogStash::Inputs::Base
     config_name "metrics"
+    milestone 3
```

Comment:
Yes we arent, but not having this make logstash display a yellow warning on boot, I have another PR #4603 that take care of that problem.